test/e2e/e2e-test/docker/simple/compat/docker-compose.yml

```diff
@@ -0,0 +1,61 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '2.1'
+
+services:
+  oap:
+    environment:
+      SW_PROMETHEUS_FETCHER: "default"
+      SW_TELEMETRY: prometheus
+    extends:
+      file: ../../base-compose.yml
+      service: oap
+
+  ui:
+    extends:
+      file: ../../base-compose.yml
+      service: ui
+    depends_on:
+      oap:
+        condition: service_healthy
+
+  provider:
+    build:
+      context: ../../../..
+      dockerfile: e2e-test/docker/Dockerfile.provider
+      args:
+        - SW_AGENT_JDK_VERSION=${SW_AGENT_JDK_VERSION:-11}
+        - DIST_PACKAGE=https://dist.apache.org/repos/dist/release/skywalking/8.3.0/apache-skywalking-apm-8.3.0.tar.gz
```

Comment:
Use archive repo, rather than release repo. We will forget the update.

Reply:
Dist package could be remote? We already have this codes?

Reply:
> Use archive repo, rather than release repo. We will forget the update.

sure, will change it when the runs are verified. 

> Dist package could be remote? We already have this codes?

huh yes, this leverages Dockerfile’s `ADD` directive, which supports `http`, we used this to test es6 and es7 dist packages but now it can be used in this scenario.